src/myrtlespeech/model/cnn.py

```diff
@@ -332,9 +332,6 @@ def forward(
 
         return acts, seq_lens
 
-    def extra_repr(self) -> str:
-        return super().extra_repr() + f", padding_mode={self.padding_mode}"
-
```

Comment:
These changes are required because the upgrade -> `pytorch==1.4.0` meant that the `padding_mode` was now being printed twice. Caught by the doctests.

Reply:
Good point - turns out we were overriding the `padding_mode` attribute in `torch.nn.Conv1d`. I think this attribute is a `>1.2` addition as it wasn't occurring previously. 

i.e. 
```python
from enum import Enum

import torch

class PaddingMode(Enum):
    """Available padding modes."""
    NONE = 0
    SAME = 1

class Foo(torch.nn.Conv1d):
    def __init__(self, padding):
        super().__init__(
            in_channels=2,
            out_channels=3,
            kernel_size=4,
            stride=1,
        )
        self.padding_mode = padding
```
```shell
>>> Foo(padding=PaddingMode.NONE)
Foo(2, 3, kernel_size=(4,), stride=(1,), padding_mode=PaddingMode.NONE)

Reply:
I've replaced our attribute `self.padding_mode -> self._padding_mode`.